.github/workflows/matomo-tests.yml

```diff
@@ -53,6 +53,8 @@ jobs:
           persist-credentials: false
           submodules: true
           path: matomo
+      - name: Install package ripgrep
+        run: sudo apt-get install ripgrep
```

Comment:
This might be ok for the CI, but I'm not sure how would people test this locally without the dependency (and installing it)?

Is there a Composer-package equivalent we could perhaps require in `require-dev`?

Did you consider any other potential approaches, e.g. using static analysis tools like [PHPStan](https://github.com/phpstan/phpstan) or [Psalm](https://github.com/vimeo/psalm)?

Reply:
@michalkleiner I checked `git grep` but it works only for current directory, so this common command can't be used for other plugins, we can add a check in the command if the desired package is installed or not and alert the user.

Reply:
@michalkleiner Added an exception if package is not installed and it also shows a link from where you can install it